lib/apiservers/engine/backends/container.go

```diff
@@ -740,7 +740,7 @@ func (c *Container) ContainerRestart(name string, seconds *int) error {
 	}
 
 	operation = func() error {
-		return c.containerStart(name, nil, false)
+		return c.containerStart(name, nil, true)
```

Comment:
I'll add verification that the IP address doesn't change on a restart -- initial testing looks like it will work fine.